src/scikit_build_core/cli/__init__.py

```diff
@@ -0,0 +1,15 @@
+from __future__ import annotations
+
+import importlib.util
+
+if not importlib.util.find_spec("click"):
+    msg = "Missing cli dependencies. Make sure scikit-build-core is installed with [cli] optional dependency"
+    raise ImportError(msg)
+
+# Load all subcommands
+from . import build, configure, dynamic_metadata, install, metadata  # noqa: F401
+
+# Load and expose the main CLI interface
+from .main import skbuild
```

Comment:
Just a comment, I wanted to re-order these to load `skbuild` first and then all subcommands, just so that there is some logical consistency, but `I001` does not allow it.